store/entity-store.html

```diff
@@ -56,15 +56,16 @@
 		},
 
 		addListener: function(entityId, token, listener) {
+			this._getToken(token).then(({cacheKey, tokenValue}) => {
```

Comment:
I'm not noticing arrow functions or destructuring being used throughout the rest of the library. Might be safer to stick to things that won't require compilation

Reply:
That's too bad. I had noticed Set, Map, and Promise but no polyfills so I figured we were using polymer build / serve already. I guess we just have an expectation that Promise is polyfilled and we tread carefully with Set and Map for IE11. I'll revert my changes.